packages/shared/src/features/onboarding/shared/DiscountTimer.tsx

```diff
@@ -41,7 +41,7 @@ const calculateTimeLeft = (
 /**
  * Sanitizes HTML string and allows only bold tags
  */
-const sanitizeMessage = (message: string): string => {
+export const sanitizeMessage = (message: string): string => {
```

Comment:
Let's move it somewhere else maybe within the onboarding scope?

Reply:
Also updated the sanitizer to allow `br` for line breaks if in case they what a specific break point.

Reply:
With that, we won't need the introduction of `subheading` since we can break the line from the `heading` now.